.gitignore

```diff
@@ -5,3 +5,7 @@ npm-debug.log
 coverage
 
 dist/
+
+# Personal make commands
+local.mk
```

Comment:
Would any of the personal commands that you're adding make sense to be in the overall Makefile? That way we don't have to worry about ignoring them and others could leverage them as well? Or are they very specific to your working environment?

Reply:
I added a `make restart-detached` command for myself, which basically just does `up-detached` after the down. I didn't get the sense that anyone really used the up-detached command, so I didn't want to clutter the Makefile with my commands. If people would use it, I'm happy to add it. What do you think?

Reply:
I think it's a pretty useful command that would be good to have available for the group in general :) One more command won't clutter it up.